src/app/shared/store/user.state.ts

```diff
@@ -462,6 +468,16 @@ export class UserState {
     this.router.navigate(['']);
   }
 
+  @Action(GetStatusForNewApplication)
+  getStatusForNewApplication({ patchState }: StateContext<UserStateModel>, { payload }: GetStatusForNewApplication): Observable<boolean> {
+    return this.applicationService
+      .getStatusForNewApplication(payload.workshopId, payload.childId)
+      .pipe(
+        tap((status: boolean) => {
+          return patchState({ isAllowNewApplication: status });
+        }));
```

Comment:
simplify it tap((status: boolean) =>patchState({ isAllowNewApplication: status }) ));